### PR TITLE
Master to Main branch

### LINKED ISSUE #206: Master/Main branches on GitHub
**GitHub master branch to main**
[On October 1st](https://github.com/github/renaming#on-october-1-2020-newly-created-repositories-will-default-to-main), GitHub will default any new repositories  `master` branches to `main`.

This change will most likely cause issues with our linked READMEs and CircleCI configurations on this repo as well as any new repo that is created.

**Potential Solutions**
We should probably be proactive and start a PR changing all instances of `master` to `main` before October 1st. This will alleviate some of the pain we will be feeling on that week.

An alternative would be to [opt out](https://github.com/github/renaming#on-october-1-2020-newly-created-repositories-will-default-to-main) to give us some time to make the changes. It will require us to make sure every repo has a default branch and/or [setting up our organization](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-the-default-branch-name-for-repositories-in-your-organization) to use master as the default branch for the time being. I think I like this approach more because then we wont need to scramble. 

This seems to be the way the industry is moving so we would probably make the change at some point. Thoughts?

### PR DESCRIPTION
**Pull request by michaelachrisco**
## Changes
1. Blanket changes references from `master` branch to `main` branch.

## Purpose
To change all branch references from master to main on October 1st, the same day GitHub will be changing all new and some existing default branches.

## Approach
By changing this repo first, we get a head start on the new branch changes that will come into effect shortly.

This will be a Work in Progress until Oct 1st. On October 1st, the WIP will be removed. 

Closes #206

### CODE REVIEW

**Comment by michaelachrisco** [comment]
If/when this is merged, is anyone opposed to switching the repos default branch from the master branch to the main branch? 

Right before the switch, I plan on making a copy of master as main and making sure both are up to date.

**Review by Basim-Elayan — APPROVED** [review]
LGTM, Nice job I'm happy to see that GitHub update in our s&p so quickly.

File: standards/branching.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ For example, if I'm (Corey) working on an bug fix task, I'll create a branch suc
 
 When work is completed and tested, make a Pull Request to the development branch. This PR requires review from another developer before it is merged.
 
-When code is ready to be locked in for demo, a Pull Request from `development` to `master` will be created and reviewed.
+When code is ready to be locked in for demo, a Pull Request from `development` to `main` will be created and reviewed.
```
[the remark this answers is not in the document]
Comment (michaelachrisco) [reply standards/branching.md:30]: Thats a good one. Let me know If you see any others! 

We can make separate issues out of them.

**Review by mwallert — APPROVED** [review]
LGTM one suggestion but looks like you covered all the `master` to `main` changes :+1:

**Review by coreyshuman — APPROVED** [review]
The most punctual update this repo has ever received!

**Pull request merged**